README.md

```diff
@@ -16,21 +16,38 @@ Follow [these](#additional-documentation-sources) instructions to enable optiona
 
 ## Installation
 
-### Arch Linux / Manjaro ([AUR](https://aur.archlinux.org/packages/wikiman/))
+### Arch Linux
+
+**Wikiman** is available in the [community repository](https://archlinux.org/packages/community/any/wikiman/) and can be installed via [pacman](https://wiki.archlinux.org/title/Pacman):
+
 ```bash
-yay -Syu wikiman
+pacman -S wikiman
 
 # Optional: Enable Arch Wiki
-yay -Syu arch-wiki-docs
+pacman -S arch-wiki-docs
 ```
-If you are running Manjaro, package `arch-wiki-docs` is not in official repositories.
-Follow [these](#installing-additional-sources) instructions to download it.
 
-Or download latest *.pkg.tar.zst* package from [Releases](https://github.com/filiparag/wikiman/releases/latest/) tab.
+Or you can download the latest *.pkg.tar.zst* package from [Releases](https://github.com/filiparag/wikiman/releases/latest/) tab.
+
 ```bash
 sudo pacman -U wikiman*.pkg.tar.zst
 ```
 
+### Manjaro
+
+Similar intructions with [Arch Linux](#arch-linux) apply.
+
+```bash
+yay -Syu wikiman
```

Comment:
I'm not sure whether keeping instructions for Manjaro using `yay` makes sense if the package got yanked from AUR and is not in their repositories. This command currently doesn't work on Manjaro.

Reply:
I'm not familiar with Manjaro, what do you suggest here?

Reply:
Sounds good to me.